src/directory/directory.js

```diff
@@ -1117,919 +1117,917 @@ const directory = {
-          },
-          {
-            title: "Set up fullstack project",
-            route: "/start/getting-started/setup",
+              'js',
+              'react',
+              'react-native',
+              'angular',
+              'vue',
+              'next',
+              'android',
+              'ios',
+              'ionic',
+              'flutter'
+            ]
+          },
+          {
+            title: 'Set up fullstack project',
+            route: '/start/getting-started/setup',
             filters: [
-              "js",
-              "react",
-              "react-native",
-              "angular",
-              "vue",
-              "next",
-              "android",
-              "ios",
-              "ionic",
-              "flutter",
-            ],
+              'js',
+              'react',
+              'react-native',
+              'angular',
+              'vue',
+              'next',
+              'android',
+              'ios',
+              'ionic',
+              'flutter'
+            ]
           },
           {
-            title: "Generate model files",
-            route: "/start/getting-started/generate-model",
-            filters: ["android", "ios", "flutter", "react-native"],
+            title: 'Generate model files',
+            route: '/start/getting-started/generate-model',
+            filters: ['android', 'ios', 'flutter', 'react-native']
           },
           {
-            title: "Integrate your app",
-            route: "/start/getting-started/integrate",
-            filters: ["android", "ios", "flutter", "react-native"],
+            title: 'Integrate your app',
+            route: '/start/getting-started/integrate',
+            filters: ['android', 'ios', 'flutter', 'react-native']
           },
           {
-            title: "Connect to the cloud",
-            route: "/start/getting-started/add-api",
-            filters: ["android", "ios", "flutter", "react-native"],
+            title: 'Connect to the cloud',
+            route: '/start/getting-started/add-api',
+            filters: ['android', 'ios', 'flutter', 'react-native']
           },
           {
-            title: "Connect API and database to the app",
-            route: "/start/getting-started/data-model",
-            filters: [
-              "js",
-              "react",
-              "angular",
-              "vue",
-              "next",
-              "ionic",
-            ],
+            title: 'Connect API and database to the app',
+            route: '/start/getting-started/data-model',
+            filters: ['js', 'react', 'angular', 'vue', 'next', 'ionic']
           },
           {
-            title: "Add authentication",
-            route: "/start/getting-started/auth",
-            filters: ["react", "angular", "vue"],
+            title: 'Add authentication',
+            route: '/start/getting-started/auth',
+            filters: ['react', 'angular', 'vue', 'ionic']
```

Comment:
This was really the only change here, so `Add authentication` would show up for ionic